src/Files.Uwp/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -46,7 +45,7 @@
                 <MenuFlyoutItem
                     x:Name="ApplyToAllOption"
                     Click="MenuFlyoutItem_Click"
-                    Tag="{helpers:ResourceString Name=ConfictingItemsDialogOptionApplyToAll.Tag}"
```

Comment:
Does this resuource exit? `ConfictingItemsDialogOptionApplyToAll.Tag`

Reply:
Does not exist